### PR TITLE
Deploy production

### PR DESCRIPTION
**Pull request by mafonso**
### Jira link

HOTT-442

### What?

I have added/removed/altered:

- [ ] Allow deployment to production on manual gate.

### Why?

I am doing this because:
- Implementing workflow defined on a team chat.
-
-

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -140,50 +142,64 @@ jobs:
...
           domain_prefix: "staging"
 
+  deploy_production:
+    docker:
+      - image: cimg/ruby:2.7-node
+    steps:
+      - cf_deploy:
+          space: "production"
+          environment_key: "production"
+          domain_prefix: ""
```
Comment (willfish) [inline .circleci/config.yml]: Isn't the domain prefix www?

Comment (mafonso) [reply .circleci/config.yml]: It is  yes!  🤦 
Which means I need to find another way of doing this.   `www`  vs `assets`

**Comment by willfish** [comment]
I think we should remove the hold on deploying to staging since we've got a filter for branches == master

**Comment by willfish** [comment]
We work a lot to make sure what we've merged has been reviewed, understood and tested and merged branches are highly stable

File: .circleci/config.yml (hunk 1 above)
Comment (mafonso) [reply .circleci/config.yml]: To be fair, this is something that I wanted to have a look at. It does not make sense to me that the origin needs to know what's its external name. 
I know why it's happening, but I think it should not be like this. The origin can and should be agnostic, IMO.

**Comment by mafonso** [comment]
> I think we should remove the hold on deploying to staging since we've got a filter for branches == master

Is that still in? Can't find it...

File: .circleci/config.yml (hunk 1 above)
Comment (willfish) [reply .circleci/config.yml]: Yeah

**Comment by willfish** [comment]
No it's not! Is my brain fart :)

**Review by willfish — APPROVED** [review]

**Pull request merged**